scripts/eurostat/health_determinants/alcohol_consumption/process.py

```diff
@@ -0,0 +1,820 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+Before running this module, run download_input_files.py script, it downloads
+required input files, creates necessary folders for processing.
+Folder information
+input_files - downloaded files (from US census website) are placed here
+output_files - output files (mcf, tmcf and csv are written here)
+"""
+import os
+import sys
+import re
+import pandas as pd
+import numpy as np
+from absl import app, flags
+
+# For import common.replacement_functions
+_COMMON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.insert(1, _COMMON_PATH)
+# pylint: disable=import-error
+# pylint: disable=wrong-import-position
+from common.replacement_functions import (_replace_sex, _replace_frequenc,
+                                          _replace_isced11, _replace_quant_inc,
+                                          _replace_deg_urb, _replace_c_birth,
+                                          _replace_citizen, _split_column)
+# For import util.alpha2_to_dcid
+_COMMON_PATH = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '../../../../'))
+sys.path.insert(1, _COMMON_PATH)
+
+from util.alpha2_to_dcid import COUNTRY_MAP
+# pylint: enable=import-error
+# pylint: enable=wrong-import-position
+
+_FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_MCF_TEMPLATE = ("Node: dcid:{pv1}\n"
+                 "{pv11}\n"
+                 "typeOf: dcs:StatisticalVariable\n"
+                 "populationType: dcs:Person{pv2}{pv3}{pv4}{pv5}"
+                 "{pv6}{pv7}{pv8}{pv9}{pv10}\n"
+                 "statType: dcs:measuredValue\n"
+                 "measuredProperty: dcs:count\n")
+
+_TMCF_TEMPLATE = (
+    "Node: E:EuroStat_Population_AlcoholConsumption->E0\n"
+    "typeOf: dcs:StatVarObservation\n"
+    "variableMeasured: C:EuroStat_Population_AlcoholConsumption->SV\n"
+    "measurementMethod: C:EuroStat_Population_AlcoholConsumption->"
+    "Measurement_Method\n"
+    "observationAbout: C:EuroStat_Population_AlcoholConsumption->geo\n"
+    "observationDate: C:EuroStat_Population_AlcoholConsumption->time\n"
+    "scalingFactor: 100\n"
+    "value: C:EuroStat_Population_AlcoholConsumption->observation\n")
+
+
+def _alcoholconsumption_by_sex_education(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Given all the CSVs have one of 2 structures (right?), instead of duplicating this code so many times, we should try to structure it as two functions that take a map and process the data.  The same might be true of other eurostat scripts as well.

This is a broader topic, lets bring it up in the next team meeting.